### PR TITLE
[IMP] stock: Add a void button box on Package Types form view

### PR DESCRIPTION
**Pull request by rousseldenis**
---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by rousseldenis** [comment]
Solves https://github.com/odoo/odoo/issues/103420

File: addons/stock/views/stock_package_type_view.xml [hunk 1]
```diff
@@ -7,6 +7,9 @@
     <field name="arch" type="xml">
         <form string="Package Type">
             <sheet>
+                <div class="oe_button_box" name="button_box">
+
+                </div>
```
Comment (MiquelRForgeFlow) [inline addons/stock/views/stock_package_type_view.xml:12]:
```suggestion
                <div class="oe_button_box" name="button_box"/>
```
cleaner :)

Comment (rousseldenis) [reply addons/stock/views/stock_package_type_view.xml:12]: Done

**Comment by rousseldenis** [comment]
@MiquelRForgeFlow @lmignon

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by lmignon — APPROVED** [review]

**Comment by Whenrow** [comment]
robodoo r+

**Pull request closed without merging**